src/apps/components/AppAlerts/useAllAppsAlert.ts

```diff
@@ -1,19 +1,41 @@
 import { useUserPermissions } from "@dashboard/auth/hooks/useUserPermissions";
-import { PermissionEnum, useAppFailedPendingWebhooksQuery } from "@dashboard/graphql";
+import {
+  AppFailedPendingWebhooksQuery,
+  EventDeliveryStatusEnum,
+  PermissionEnum,
+  useAppFailedPendingWebhooksQuery,
+} from "@dashboard/graphql";
 import { useMemo } from "react";
 
-const requiredPermissions = [PermissionEnum.MANAGE_APPS];
-
+type Webhook = NonNullable<
+  NonNullable<AppFailedPendingWebhooksQuery["apps"]>["edges"][0]["node"]["webhooks"]
+>[0];
 interface FailedWebhooksCount {
   hasFailed: boolean;
   hasPending: boolean;
 }
 
+const requiredPermissions = [PermissionEnum.MANAGE_APPS];
+
 const defaultFailedWebhooksInfo: FailedWebhooksCount = {
   hasFailed: false,
   hasPending: false,
 };
 
+const hasFailedCheck = (webhook: Webhook) =>
+  webhook.failedDelivers && webhook.failedDelivers?.edges?.length > 0;
+const hasPendingCheck = (webhook: Webhook) => {
```

Comment:
You are checking here if failed attempt occurs, so the name of the function does not do what it says. It's something like "hasFailedAttempt"